scss/_navbar-offcanvas.scss

```diff
@@ -81,12 +81,14 @@
     }
   }
 
+  .nav {
+    max-height: 85vh;
+    flex-wrap: initial;
+  }
+
   .navbar-nav {
     width: 100%;
     overflow-y: auto;
-    .nav-link {
-      font-weight: 500;
-    }
```

Comment:
Keep this and add `.nav-link {font-weight: 700;}` to the media-breakpoint-up(lg) query. Then the off-canvas styles won't change from how they were before. 

I would add this as suggestions, but GitHub isn't allowing me. :(